vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/vaadin-grid-flow-selection-column.js

```diff
@@ -23,6 +23,14 @@ import { GridColumnElement } from '@vaadin/vaadin-grid/src/vaadin-grid-column.js
     static get properties() {
       return {
 
+        /**
+         * set auto-width is true
```

Comment:
This should be the description of the property, not about setting it to some value. Maybe just copy the first line from https://github.com/vaadin/vaadin-grid/blob/master/src/vaadin-grid-column.html#L548 instead (no need to copy the whole description as it's quite verbose)

Reply:
yes, I corrected it to "Automatically sets the width of the column based on the column checkbox when this is set to `true`."

Reply:
corrected it to ...column content...